common/backoff/retrypolicy_test.go

```diff
@@ -47,7 +47,8 @@ type (
 // maximum interval of 10 seconds. Keep in mind that there is a random jitter in these times, so they are not exactly
 // what you'd expect.
 func ExampleExponentialRetryPolicy_WithMaximumInterval() {
-	rand.Seed(42)
+	jitterRand.Store(rand.New(rand.NewSource(42)))
```

Comment:
Looks like this is getting unnecessarily complicated just to replace this call. How about we fix this and make it a regular test (right now it's an example test - https://go.dev/blog/examples). This way you won't need deterministic jitter values to make it pass.